docs/multiple-tests/pattern-vulnerability/results.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<checkstyle version="1.5">
+    <file name="Gemfile.lock">
+        <error source="vulnerability" line="4" message="Insecure dependency puma@6.3.0 (CVE-2023-40175: HTTP request smuggling when parsing chunked transfer encoding bodies and zero-length content-length headers) (update to ~\u003e 5.6.7, \u003e= 6.3.1)" severity="error" />
+    </file>
+</checkstyle>
```

Comment:
This is the result file of running the patterns in `patterns.xml`? If so, should this be included in the PR or this is just an example?

Reply:
This is the "result" of running the tool on the `Gemfile.lock`. This is used for testing.

Reply:
@Afonsobli you can read more about it here: https://github.com/codacy/codacy-plugins-test?tab=readme-ov-file#multiple-test-definition